index.html

```diff
@@ -40,103 +41,102 @@
                     <span class="bar"></span>
                 </div>
             </nav>
-            
         </header>
         <section class="welcome">
             <div class="welcome-section-container">
-                <div class="">
-                    <p>"Welcome to the Wild East!"</p>
+                <div>
+                    <p class="akwaaba">"Welcome to the Wild East!"</p>
                     <p class="event-name">SAFARI RETREAT IN THE SERENGETTI 2023</p>
```

Comment:
- [ ] This is the page title, for good practice, the title goes in the `hx` tag. Please kindly use the `hx` tag for all the title and not the `p` tag, the `p` tag is for paragraph 🙃 🙃